manager/src/test/java/com/redhat/service/bridge/manager/dao/ProcessorDAOTest.java

```diff
@@ -164,7 +165,7 @@ public void findByBridgeIdAndCustomerId() {
     @Test
     public void findByBridgeIdAndCustomerId_noProcessors() {
         Bridge b = createBridge();
-        ListResult<Processor> listResult = processorDAO.findByBridgeIdAndCustomerId(b.getId(), TestConstants.DEFAULT_CUSTOMER_ID, 0, 100);
+        ListResult<Processor> listResult = processorDAO.findByBridgeIdAndCustomerId(b.getId(), TestConstants.DEFAULT_CUSTOMER_ID, new QueryInfo(0, 100));
```

Comment:
Two general comments on the test, even if they are not related to the PR per se.

- probably you should make Bridge b a instance variable for the test, this would remove a parameter from create processor and make the code clearer

- the assertion assertThat(listResult.getPage()).isEqualTo(1L) is very verbose and exposes lots of implementation details on how to test that the page of listResult is 1. I would replace it with a custom assertion like pageForListResultIs(1). The same for other assertions. In general I would try to make the test very terse, just providing the information that is specific for the test and hide all the other details.